### DISCUSSION TITLE
Plex for Windows x64 (Preview) - Working Fine with PGMA

### DISCUSSION THREAD

**Discussion by jfreiman** [CodyBerenson/PGMA-Modernized#189] [General]
I just wanted other PGMA users to know that the Preview of Plex Server x64 is working fine with PGMA,
While others in the Plex Forums are stating that the software is performing faster than the x32 bit version, I haven't seen any performance increase.
I have noticed that Plex crashes less frequently - purely non-scientific opinion. I can say that is certainly isn't crashing MORE!

I just wanted to let other users know that this [Preview is available on the Plex Forums](https://forums.plex.tv/t/preview-plex-media-server-for-windows-64-bit-version/801401) and also that it appears to work fine with the PGMA.

Note: I do have Windows Security "Controlled Folder Access" enabled in the Ransomware section. That means I did have to give permissions to the Plex Media Server and Plex Media Scanner (x64 versions) apps to have access to my collections.

**Comment by JPH71** [comment]
Cheers for this...

I have noticed a lot of crashing on my non porn server when the transcoder
runs..... read that if you clean bundles, empty trash and optimise.. that
this helps with transcoder issues - if any of you are suffering from
this....
